server/src/infra/repositories/access.repository.ts

```diff
@@ -156,4 +157,15 @@ export class AccessRepository implements IAccessRepository {
       });
     },
   };
+
+  person = {
+    hasOwnerAccess: (userId: string, personId: string): Promise<boolean> => {
+      return this.personRepository.exist({
```

Comment:
I don't do this other places, but I think there is a variation you can use `existBy` the is basically just the `where` object.